common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                            "State the academic policy for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain forum participation expectations"),
+                        "long_description": _(
+                            "State the expectation for learner forum participation for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Provide forum etiquette guidelines"),
+                        "long_description": _(
+                            "State the guidelines for forum use and ediquette for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain instructor/TA involvement"),
+                        "long_description": _(
+                            "Explain the role and level of involvement that instructors and course TAs for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Set Section Release Dates"),
+                        "short_description": _("Explain course material release schedule"),
                         "long_description": _(
-                            "Specify the release dates for each Section in your course. Sections become visible to "
-                            "students on their release dates."
+                            "State the course material release schedule and frequency of updates for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Communicate any changes to released content"),
+                        "long_description": _(
+                            "Agree to clearly communicate to learners when any changes have been made to released content"
```

Comment:
Identify how and when you will communicate changes that affect released course content to learners.